ci/deps/azure-37-locale.yaml

```diff
@@ -5,8 +5,7 @@ dependencies:
   - python=3.7.*
 
   # tools
-  # Cython pin for https://github.com/pandas-dev/pandas/issues/34704
```

Comment:
do we want to revert this pin?

Reply:
Yea that pin was a quick fix for CI but not needed if you remove these div methods

Reply:
i c ok